Tests/MongoKittenTests/CRUDTests.swift

```diff
@@ -1,55 +1,180 @@
-//import NIO
-//import MongoKitten
-//import XCTest
-//
-//let dbName = "test"
-//
-//class RemoteDatabaseCRUDTests : XCTestCase {
+import NIO
+import MongoKitten
+import XCTest
+
+let dbName = "KittenTest"
+
+class CrudTests : XCTestCase {
+  struct DummyAccount: Codable {
+    static let collectionName = "DummyAccounts"
+
+    let _id: ObjectId
+    var name: String
+    var password: String
+    var age: Int
+    
+    init (Name name: String, Password password: String, Age age: Int) {
```

Comment:
`init(named name: String, password: String, age: Int)`
Don't use uppercased for variables and labels.